scripts/patch-versions/util.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const path = require("path");
+
+const getDirsFromPath = (dirsPath) => fs.readdirSync(dirsPath).filter(dirName => isDirectory(dirsPath, dirName)).map(dirName => path.resolve(dirsPath, dirName));
+
+const isDirectory = (dirPath, fileOrDirName) => fs.statSync(path.join(dirPath, fileOrDirName)).isDirectory();
+
+const hasPackageJSONFile = (dirPath) => fs.existsSync(path.join(dirPath, 'package.json'));
+
+const readFilePromise = (file) => new Promise((resolve, reject) => fs.readFile(file, 'utf-8', (err, data) => err ? reject(err) : resolve(data)));
+
+const getPackageDirectories = (packagesDir) => getDirsFromPath(packagesDir).filter(dir => hasPackageJSONFile(dir))
+
+const getPackagesVersions = async (packageDirs, packageFileName) => packageDirs.reduce(async (acc, dir) => {
+    const packageFile = path.resolve(dir, packageFileName);
+    const file = JSON.parse(await readFilePromise(packageFile));
+    acc.then(obj => obj[file.name] = file.version);
+    return acc;
+}, Promise.resolve({}));
+
+const updatePackageJson = (filesData, packageVersionsMap) => {
+    const { packageJsonPath, packageJsonData } = filesData;
+    const dep = packageJsonData.dependencies || [];
+    const devDep = packageJsonData.devDependencies || [];
+
+    console.log('Package JSON file path: ', packageJsonPath)
```

Comment:
```suggestion
    console.log(`Updating ${packageJsonPath}`)
```
Is this a string?

Reply:
f169a8a5f2791b39243b3f8918db17305b909da4